packages/deployment-service/src/functions/pipeline/task-runner.ts

```diff
@@ -24,6 +24,7 @@ const deleteMessage = (ReceiptHandle: string): Promise<void> => {
 }
 
 const failure = async (task: TaskEntity, receiptHandle, error?: Error): Promise<void> => {
+  console.log(`Task [${task.functionName}] failed.`)
```

Comment:
You know you can use the `@reapit/utils` logger - defaults to console locally and logs to CloudWatch remotely. Might be good for debugging generally

Reply:
sure I'll change that, thanks!